config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml

```diff
@@ -233,6 +233,45 @@ presubmits:
 
     annotations:
       testgrid-create-test-group: 'true'
+  - name: pull-kubernetes-e2e-gce-serial
```

Comment:
Can we add some "storage" word to the name?

Reply:
done, renamed it to `pull-kubernetes-e2e-gce-storage-disruptive`